locale/de/LC_MESSAGES/django.po

```diff
@@ -262,7 +262,7 @@ msgid "Can Remove Organization"
 msgstr "Kann die Organisation entfernen"
 
 #: accounts/models.py:70
-msgid "Can View Experimenter"
+msgid "Can View Researcher"
 msgstr "Kann den Experimentator sehen"
 
```

Comment:
(the translations should end up being unnecessary if we don't make the changes to the template text / permission names as noted above)